### PR TITLE
Adding DDS to quickstart

### PR DESCRIPTION
**Pull request by rjriel**

### CODE REVIEW

File: node/src/serve.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ if (validProductTypes.indexOf(API_PRODUCT_TYPE) < 0) {
   process.exit(-1)
 }
 
-const html = readFileSync(`../html/${API_PRODUCT_TYPE}.html`)
+const html = readFileSync(`../html/${API_PRODUCT_TYPE === 'deposit_switch' ? 'dds' : API_PRODUCT_TYPE}.html`)
```
Comment (chetvil) [inline node/src/serve.js:16]: why not to name html deposti_switch.html?

Comment (rjriel) [reply node/src/serve.js:16]: that makes sense, will make the change

File: c-sharp/Controllers/DdsController.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+using System.Text.Json;
```
Comment (chetvil) [inline c-sharp/Controllers/DdsController.cs:1]: the file name better DDSController?

File: c-sharp/Controllers/DdsController.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    [Route("{token}")]
+    [HttpGet]
+    public async Task<string> Get(string token)
+    {
+      var accessTokenResponse = await _citadel.GetAccessToken(token);
+      var parsedResponse = JsonDocument.Parse(accessTokenResponse);
+      var accessToken = parsedResponse.RootElement.GetProperty("access_token").GetString();
+
+      return await _citadel.GetDdsByToken(accessToken);
```
Comment (chetvil) [inline c-sharp/Controllers/DdsController.cs:22]: GetDDSByToken?

File: html/deposit_switch.html [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+          }).then((r) => r.json())
+          console.table(response)
+          return response
+        }
+
+        // sends a public_token to quickstart back end so the quickstart
+        // back end can exchange it with Citadel for an access_token
+        const getDdsData = async (public_token) => {
+          const response = await fetch(
+            apiEnpoint + `getDdsData/${public_token}`,
```
Comment (chetvil) [inline html/deposit_switch.html:169]: getDDSData

File: node/src/citadel.js [hunk 5]
```diff
@@ -187,6 +202,7 @@ const sendRequest = async (endpoint, { body = undefined, method = "POST" }) => {
 }
 
 export {
+  getDdsByToken,
```
Comment (chetvil) [inline node/src/citadel.js:205]: getDDSByToken?

File: ruby/app/models/Citadel.rb [hunk 6]
```diff
@@ -80,6 +80,14 @@ def self.completeFasFlowByToken(access_token, first_micro, second_micro)
     sendRequest('refresh/tasks/', body, "POST")
   end
 
+  def self.getDdsByToken(access_token)
```
Comment (chetvil) [inline ruby/app/models/Citadel.rb:83]: same here DDS

**Review by chetvil — APPROVED** [review]

**Pull request merged**